leap/leap_test.rb

```diff
@@ -15,26 +18,44 @@ def leap?
 
 class YearTest < Minitest::Test
   def test_leap_year
-    assert Year.leap?(1996), 'Yes, 1996 is a leap year'
+    assert_equal true, Year.leap?(1996)
   end
 
   def test_non_leap_year
     skip
-    refute Year.leap?(1997), 'No, 1997 is not a leap year'
+    assert_equal false, Year.leap?(1997)
   end
 
   def test_non_leap_even_year
     skip
-    refute Year.leap?(1998), 'No, 1998 is not a leap year'
+    assert_equal false, Year.leap?(1998)
   end
 
   def test_century
     skip
-    refute Year.leap?(1900), 'No, 1900 is not a leap year'
+    assert_equal false, Year.leap?(1900)
   end
 
   def test_fourth_century
     skip
-    assert Year.leap?(2400), 'Yes, 2400 is a leap year'
+    assert_equal true, Year.leap?(2400)
+  end
+
+  def test_Y2K
+    skip
+    assert_equal true, Year.leap?(2000)
+  end
+
+  # Problems in exercism evolve over time,
+  # as we find better ways to ask questions.
+  # The version number refers to the version of the problem you solved,
+  # not your solution.
+  #
+  # Define a constant named VERSION inside of Leap.
+  # If you are curious, read more about constants on RubyDoc:
+  # http://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/constants.html
+  def test_bookkeeping
+    skip
+    assert_equal 2, Year::VERSION
```

Comment:
Pff, I thought I'd corrected that. It seems as each time you generate the tests, it gets incremented and I must have lost track of resting it. Will sort it out.